lambda/lambda-with-business-wallet/README.md

```diff
@@ -0,0 +1,89 @@
+# MetaKeep Lambda - Simple Voting Application
```

Comment:
folder name should just be `business-wallet`

Reply:
done